python/F3DPythonBindings.cxx

```diff
@@ -130,7 +130,12 @@ PYBIND11_MODULE(pyf3d, module)
     .def_property_readonly("channel_type", &f3d::image::getChannelType)
     .def_property_readonly("channel_type_size", &f3d::image::getChannelTypeSize)
     .def_property("content", getImageBytes, setImageBytes)
-    .def("compare", &f3d::image::compare)
+    .def("compare",
+      [](const f3d::image& self, f3d::image& other, double threshold)
+      {
+        double error = 0;
```

Comment:
BTW 0 is too low, should be either 1e-14 or 0.04

Reply:
isn't it just a variable to serve as an out-param?

Reply:
indeed, misunderstood that.